app/models/location.rb

```diff
@@ -0,0 +1,15 @@
+# the model representing a location of interest
+class Location < ApplicationRecord
+  include Timeable
+
+  validates :name, :location_longitude, :location_latitude, presence: true
```

Comment:
location_latitude and location_longitude are checked twice for presence
maybe better: solution like timable with e.g. "include Locatable" which generates this (but probably not in this PR). What do you think? Should we open a new issue?

Reply:
we removed the duplicate checks but did not introduce a "include Locatable"